distribution/server/src/assemble/LICENSE.bin.txt

```diff
@@ -338,11 +338,12 @@ The Apache Software License, Version 2.0
     - io.prometheus-simpleclient_tracer_otel-0.16.0.jar
     - io.prometheus-simpleclient_tracer_otel_agent-0.16.0.jar
  * Prometheus exporter
-    - io.prometheus-prometheus-metrics-config-1.3.3.jar
-    - io.prometheus-prometheus-metrics-exporter-common-1.3.3.jar
-    - io.prometheus-prometheus-metrics-exporter-httpserver-1.3.3.jar
-    - io.prometheus-prometheus-metrics-exposition-formats-1.3.3.jar
-    - io.prometheus-prometheus-metrics-model-1.3.3.jar
+    - io.prometheus-prometheus-metrics-config-1.3.4.jar
```

Comment:
is this part also included?

Reply:
yes, `Check binary licenses` job  will check this license.
https://github.com/apache/pulsar/actions/runs/12407619199/job/34645219668?pr=23756